website/pages/blog/announcing-graphql-network-inspector.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Announcing GraphQL Network Inspector
+tags: [graphql, graphql-network-inspector]
+authors: [warrenjday]
+date: 2023-02-10
+description: A better way to debug network traffic with GraphQL
+image: /blog-assets/announcing-graphql-network-inspector/cover.png
+thumbnail: /blog-assets/announcing-graphql-network-inspector/thumbnail.png
+---
+
+GraphQL Network Inspector is a dedicated devtools extension which allows you to inspect and debug
+GraphQL queries and mutations from network traffic.
+
+The extension has seen great adoption since its initial release and now I'm excited to announce it's
+new home as part of The Guild.
+
+You can check out the open source extension and try it out today.
+[GraphQL Network Inspector](https://github.com/warrenday/graphql-network-inspector)
+
+## Why GraphQL Network Inspector?
+
+If you've ever worked with GraphQL on the client, you'll know that it can be a real pain to debug
+using the standard network tab. The standard devtools are designed for REST APIs and don't provide
+clear visibility into the individual GraphQL queries and mutations being sent.
+
+With every request you'll see `POST /graphql`. This makes it difficult to distinguish requests
+without opening the request details and inspecting the body.
+
+GraphQL Network Inspector solves this problem by providing a dedicated tab for GraphQL requests and
+displaying each query in a readable format.
+
+![Compare](/blog-assets/announcing-graphql-network-inspector/request-compare.png)
+
+## Features Overview
+
+The great thing about a dedicated inspector is we can provide much more specific functionality
+around GraphQL. Here are some of the features we've added to the extension:
+
+### Request Overview
+
+The request overview tab provides a simple overview of all queries and mutations. Here you can see
+queries by name, distinguish between queries and mutations and even see total errors returned for
+each request.
+
+![Request Overview](/blog-assets/announcing-graphql-network-inspector/request-overview.png)
+
+### Request Details
+
+Clicking into a request will show similar details to the standard inspector. But here we have a
+better breakdown for each element of the request including; headers, request query with variables
+and response.
+
+![Request Details](/blog-assets/announcing-graphql-network-inspector/request-details.png)
+
+### Copy Details to GraphQL Playground
```

Comment:
Does it support only GraphQL Playground, or others as well (like graphiql or apollo studio)?

Reply:
The format of code on copy is designed for playground, although there is no reason it wouldn't work with other tools.

The end goal here is to have an "open in playground" button, which preloads the UI with the query. Although I'm not sure if it's possible to do that through query string with playground.